src/Corax/Queries/MemoizationMatch.Erasure.cs

```diff
@@ -37,21 +41,31 @@ public int AndWith(Span<long> buffer, int matches)
         {
             return _functionTable.AndWithFunc(ref this, buffer, matches);
         }
+        
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public Span<long> FillAndRetrieve()
+        {
+            return _functionTable.FillAndRetrieveFunc(ref this);
```

Comment:
Please check the warning here

Reply:
Yes, wanted to talk with you on that. Need to figure out how to fix that, not actually sure :-)